checks/sast.go

```diff
@@ -19,6 +19,8 @@ import (
 	"github.com/ossf/scorecard/checker"
 )
 
+var sast_tools map[string]bool = map[string]bool{"github-code-scanning": true, "sonarcloud": true}
```

Comment:
nit: camelCase for variable sastTools

Reply:
Can you think of any others to add here ?

Reply:
Some actions/apps that can be found [here](https://github.com/marketplace/category/security) could make sense to check for, but it would be hard to create an exhaustive list. 

Instead of checking for specific apps another approach could be to check if the app that created the check is categorized under security in the marketplace. However this might then be a bit too inclusive, since not every app listed is a SAST tool